src/material/list/list-item-sections.ts

```diff
@@ -35,8 +35,27 @@ export class MatListItemTitle {
   host: {'class': 'mat-mdc-list-item-line mdc-list-item__secondary-text'},
   standalone: true,
 })
-export class MatListItemLine {
+export class MatListItemLine implements AfterViewInit {
   constructor(public _elementRef: ElementRef<HTMLElement>) {}
+
+  ngAfterViewInit() {
+    this._addTitleToTruncatedText();
+  }
+
+  // In case text overflow is triggered and ellipsis is applied, adds role='tooltip' and title
+  // attribute to container so visual users can see the full text on hover
+  private _addTitleToTruncatedText() {
```

Comment:
This won't work if the text changes and I don't think that we would want to poll the DOM for it. Given that users project the `MatListItemLine` themselves, it might be better to just update our a11y guidelines to mention adding `role="tooltip"` for longer text.